website/docs/sql-function-glossary/sql-aggregate-functions/sql-min.md

```diff
@@ -0,0 +1,58 @@
+---
+id: sql-min 
+title: SQL MIN 
+description: The MIN aggregate function allows you to compute the minimum value from a column or across a set of rows for a column.
+slug: /sql-function-glossary/sql-min
+---
+
+<head>
+    <title>Working with SQL MIN</title>
+</head>
+
+SQL MIN, MAX, SUM… the aggregate functions that you’ll live and die by as an analytics practitioner. In this post, we’re going to unpack the SQL MIN function, how to use it, and why it's valuable in data work.
+
+To get into it, the MIN aggregate function allows you to compute the minimum value from a column or across a set of rows for a column. The results from the MIN function are useful for understanding the distribution of column values and determining the first timestamps of key events.
+
+## How to use the MIN function in a query
+
+Use the following syntax in a query to find the minimum value of a field:
+
+`min(<field_name>)`
```

Comment:
Is <field_name> our standard for referencing variables, or would we use {{ }} jinja brackets?  I'm good either way, just want to make sure we're following convention

Reply:
Yeah I use `<field_name>` across all the glossary pages; jinja would be confusing IMO since it wouldn't actually "work"